packages/aws_common/lib/src/http/aws_http_client.dart

```diff
@@ -131,8 +131,12 @@ abstract class AWSBaseHttpClient extends AWSCustomHttpClient {
     responseProgressCompleter.setSourceStream(operation.responseProgress);
     operation.operation.then(
       (resp) async {
-        resp = await transformResponse(resp);
-        completer.complete(resp);
+        try {
```

Comment:
@dnys1 double-checking this change makes sense? Without this, exceptions thrown in `transformResponse` weren't getting caught as expecting when await operation `.response`.